### PR TITLE
improve docs for no_proxy

### PR DESCRIPTION
**Pull request by mythi**
/cc @nolancon

### CODE REVIEW

File: cmd/operator/README.md [hunk 1]
```diff
@@ -60,10 +60,17 @@ spec:
...
-      value: 127.0.0.1,localhost,.example.com,10.0.0.0/8,.svc
+      value: 127.0.0.1,localhost,.example.com,10.0.0.0/8,.svc,.svc.cluster.local
     ...
 ```
 
+**Note:** To build clusters using `kubeadm` with the right `no_proxy` settings from the very beginning,
+set the cluster service names to `$no_proxy` before `kubeadm init`:
+
+```
+$ export no_proxy=$no_proxy,.svc,.svc.cluster.local
```
Comment (rojkov) [inline cmd/operator/README.md:71]: That's a very good tip! Could you please add it also to https://github.com/intel/intel-device-plugins-for-kubernetes/tree/master/cmd/fpga_admissionwebhook#pre-requisites?

Comment (mythi) [reply cmd/operator/README.md:71]: done!

**Review by rojkov — APPROVED** [review]

**Comment by rojkov** [comment]
I can't find any restart button in jenkins. Could somebody trigger the build one more time?

**Comment by mythi** [comment]
@rojkov restarted and it's passing now

**Review by rojkov — APPROVED** [review]
Thanks!

**Pull request merged**